### PR TITLE
RFC: lift ambiguity on a custom implementation of cached_property

### PR DESCRIPTION
**Pull request by neutrinoceros**
## PR Summary
In response to one of my own, misguided, TODO comments from last year.

### CODE REVIEW

**Review by matthewturk — APPROVED** [review]
seems fine -- pretty no-op, right?

**Comment by neutrinoceros** [comment]
Hopefully

File: yt/visualization/volume_rendering/old_camera.py [hunk 1]
```diff
@@ -726,7 +725,6 @@ def _render(self, double_check, num_threads, image, sampler):
         image = self.finalize_image(image)
         return image
 
-    @cached_property
```
Comment (neutrinoceros) [inline yt/visualization/volume_rendering/old_camera.py:729]: Note : this should never need to be cached. A module, once imported, is part of the global state and behaves like a singleton.
That said, I have _no_ idea why we'd want to hide pyplot behind a property, I suspect this was done with good intentions but a flawed understanding of how imports work.
I may refactor this in a future PR.

Comment (neutrinoceros) [reply yt/visualization/volume_rendering/old_camera.py:729]: update: apparently *I* added this decorator around the same time I added the flawed comment I'm fixing here. I can now confidently assert that the original author didn't know what he was doing :)

**Pull request merged**